### ISSUE TITLE
tweak UI variable language

### ISSUE THREAD

**Issue by graybeal** [metadatacenter/phs-gdc#38]
In the UI, we have a little confusion stemming from word choices. This is fairly easily fixed, here is my suggestion:

* Select Index Variable -> Select Locations
* Select the variable that will link your data to Data Commons -> Select locations to link your data to Data Commons
* Variable -> Location Type
* Select a variable from the list -> Select a location type from the list
* Variable values -> Entering location values
* Enter values by hand -> Enter location values by hand
* Use all values from selected locations -> Use all values from selected region (US states)
* Create box label (for entering values by hand): 'Location list' 
* Change box label (for using all values from selected location): 'Location' -> 'Selected states'
* Change default label inside drop-down menu: Location -> States
* Select locations (US states) -> Select region (US states)

Arguably it will be clearer the first time if there is no default radio button selected, and the box at the bottom isn't there. But it's mildly tricky to make obvious that the next step is to choose one of the radio buttons. (Maybe if we want to remove the box, then make the header of the radio selector section "Entering location values (choose one)"?)

Also, in box 2 suggest changing the following:
* Select DC Variables -> Select Data Variables
* Select the Data Commons variables you want to retrieve values from -> Select Data Commons variables to retrieve values from
* Search for variables by name or browse the hierarchy -> Select from the hierarchy or begin typing variable name
* Delete 'Note that' , begin with 'Many variables…'

![Screen Shot 2021-04-23 at 5.15.44 PM.png](https://images.zenhubusercontent.com/5cc9bdc26c1fd80bacaab82a/f1a0cb18-5383-44cb-ba59-2e7aae904bb5)

**Comment by marcosmro** [comment]
Those suggestions look good to me. Just two comments:
- We should find an alternative to "Select locations to link your data to Data Commons." We are querying Data Commons. I think that saying "link your data" might be misleading.
- With radio buttons, it's a good usability/design practice to have the first, and most commonly used choice selected by default. If we decide to unselect all choices by default, a radio button with the current two choices wouldn't be appropriate. A checkbox wouldn't be appropriate either, since their choices are not mutually exclusive. A valid solution would be to include an additional choice labeled "None" as the option selected by default in the radio button. I don't like this alternative much. Another alternative (which I prefer) would be to remove the radio button group and show by default both the "Select locations" dropdown the "Enter values" text box. The user wouldn't have to click on the radio button. He/she would just have to either enter the values into the text box or select one or several locations in the dropdown.

**Comment by graybeal** [comment]
Yes. How about "Select locations we can use to look up data in Data Commons."

I'm not in love with showing both boxes, but it's the best of the options so far I agree.  How about a checkbox that says "Use all values from entered US states", and the default is not checked?  Checking the box changes the provided box and example. The title for the section is "Enter location values".

**Comment by jvendetti** [comment]
@marcosmro - I'm not certain I follow your proposal with regard to removal of the radio buttons. When you say:

> Another alternative (which I prefer) would be to remove the radio button group and show by default both the "Select locations" dropdown the "Enter values" text box. The user wouldn't have to click on the radio button. He/she would just have to either enter the values into the text box or select one or several locations in the dropdown.

It sounds like you want the (recently renamed) "Location list" text area and "States" select box to appear in an enabled state simultaneously? It also sounds like you want to keep the paradigm that it's only valid to use one or the other component to narrow the query results? If so, what is your expectation if the the end user happens to enter something in both components?

With both components visible and enabled, I can imagine a thought process on the part of an end user where they might do something like:

* Select county for the location type
* Enter "Santa Clara, CA" in the in the location list text area
* Select Arizona in the states select box

In other words, execute a query that includes Santa Clara, CA, and all counties in the state of Arizona, aggregating my results by county.

**Comment by marcosmro** [comment]
Right, I agree. It's better to keep the radio buttons as they are, or think about a better alternative.
